Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Submissions/SubmitSubmissionValidationService.cs

```diff
@@ -82,8 +82,14 @@ public class SubmitSubmissionValidationService : ISubmitSubmissionValidationServ
             return ValidationResult.Invalid(ValidationMessages.Submission.UserHasNotProcessedSubmissionForProblem, problemId);
         }
 
+        if (submitSubmissionServiceModel.ByteContent != null &&
+            problem.SourceCodeSizeLimit < submitSubmissionServiceModel.ByteContent.Length)
+        {
+            return ValidationResult.Invalid(ValidationMessages.Submission.SubmissionFileTooBig, problemId);
+        }
+
         if (submitSubmissionServiceModel.StringContent != null &&
-            problem.SourceCodeSizeLimit < submitSubmissionServiceModel.StringContent.Length)
+            problem.SourceCodeSizeLimit < Encoding.ASCII.GetBytes(submitSubmissionServiceModel.StringContent).Length)
```

Comment:
Why Encoding.ASCII

Reply:
It is a mistake. Changed to UTF8 encoding.